### ISSUE TITLE
EMT21: "replicate" in soft_sediment sampling sheet is still formatted as "int" needs to be changed to "str" just like all the other sheets

### ISSUE THREAD

**Issue by cymon** [emo-bon/observatory-profile#33]
Processing observatory_id='EMT21' - sampling_strategy='soft_sediment' - sheet_type='sampling'
[Sample sheet link:](https://docs.google.com/spreadsheets/d/1sBB0x6h-prnUHMcOfms_7qSqTn6zeGVriitTysceoT0/gviz/tq?tqx=out:csv&sheet=sampling)

ValidationError: 1 validation error for Model
replicate
  Input should be a valid string [type=string_type, input_value=1, input_type=int]
    For further information visit https://errors.pydantic.dev/2.8/v/string_type

**Comment by kmexter** [comment]
I am not sure what the problem is here....all the values in the replicate column look ok

**Comment by isanti** [comment]
this is fine.
But, I see that replicate has changed and the input can also be a string:
1
2
3
4
blank_1
etc etc

**Comment by kmexter** [comment]
yes, replicate unfortunatley needs to account for blanks as well, so it cannot just be a number
